webapp/components/mfa/mfa_controller.jsx

```diff
@@ -5,38 +5,48 @@ import React from 'react';
 import {FormattedMessage} from 'react-intl';
 import {browserHistory, Link} from 'react-router/es6';
 
+import $ from 'jquery';
 import logoImage from 'images/logo.png';
 
 export default class MFAController extends React.Component {
     componentDidMount() {
         if (window.mm_license.MFA !== 'true' || window.mm_config.EnableMultifactorAuthentication !== 'true') {
             browserHistory.push('/');
         }
+
+        $('body').attr('class', 'sticky');
+        $('#root').attr('class', 'container-fluid');
```

Comment:
Can we do this without jquery and preferably without modifying classes outside of the render method?